docs/kit/logistics/kit-testing.md

```diff
@@ -0,0 +1,221 @@
+This allows for the removal of the manual collation step.
+
+All the automated test scripts are included in the [kit-test-scripts package](https://github.com/srobo/kit-test-scripts) the wheel file can be downloaded from [the latest release](https://github.com/srobo/.
+kit-test-scripts/releases/latest/).
+
+
+Item | Testing Step
+--- | ---
+Brain Board | [Brain Board test below](#brain-board)
+Power Board | [Power Board test below](#power-board)
+Motor Board | [Motor Board test below](#motor-board)
+Servo Board | [Servo Board test below](#servo-board)
+Arduino | [Arduino test below](#arduino)
+USB Webcam | [Camera test below](#camera)
+Battery (LiPo) | Storage charged and manually inventoried
+Battery Charger | Storage charged and manually inventoried
+Charger Supply | Storage charged and manually inventoried
+Battery bag | Manually inventoried
+USB Memory Stick | Count & wipe
+USB A to B cable | Count and cable test
+Micro USB cable | Count and cable test
+USB Hub | Count only
+CamCon (7.5mm) | Count only
+MiniCamCon (5mm) | Count only
+MicroCamCon (3.81mm) | Count only
+Screwdriver | Count only
+Screw shields | Count only
+
+## Board Tests
+
+### Brain Board
+
+Here we are testing that the KCH power circuit and that the Pi can boot and execute robot code.
+
+For this test you will need:
+
+- A 12 volt power supply with adapter to Camcon.
+- A USB stick
+
+```python
+from pathlib import Path
+import RPi.GPIO as GPIO
+asset_file = Path("/proc/device-tree/hat/custom_0")
+if asset_file.exists():
+    asset = asset_file.read_text()
+    # append code to seen file
+    with open("tested_KCHs.txt", "a") as f:
+        f.write(asset)
+        f.write("\n")
+
+    GPIO.setmode(GPIO.BCM)
+    GPIO.setup([10, 23, 18], GPIO.OUT, initial=GPIO.HIGH)
+else:
+    raise RuntimeError("No KCH found")
+```
+
+Test steps:
+
+1. Copy the above block of code into a file called `robot.py`. This should be the only file on the USB stick.
+2. Check that the Pi has an SD card inserted. Otherwise use an SD card from another Pi.
+3. Plug the USB stick into the Pi.
+4. Connect the 12V power supply to the brain board's Camcon connector.
+5. 3 Green LEDs on the Camcon end of the board will light.
+    - If they don't light immediately this board has failed, go to step 8.
+6. The 5 boot LEDs will light in order.
+    - If the first LED has not lit within 20 seconds this board has failed, go to step 8.
+    - If all 5 LEDs have not lit within approximately a minute this board has failed, go to step 8.
+7. The OK LED should then light a few colours before stabilising on green.
+    - If the LED stabilises on red this board has failed, go to step 8.
+    - If the LED has not lit within 20 seconds try replugging the USB stick. If it continues to not light this board has failed, go to step 8.
+8. Remove board making a separate pile for boards that failed.
```

Comment:
Question: Should we copy the `log.txt` off too, to track _why_ it failed?

Reply:
No, the test is easy to reproduce when it comes to fix and is awkward to match the log to the board.